.travis/pulp-operator-check-and-wait.sh

```diff
@@ -33,6 +33,8 @@ if command -v kubectl > /dev/null; then
   KUBECTL=$(command -v kubectl)
 elif [ -x /usr/local/bin/kubectl ]; then
   KUBECTL=/usr/local/bin/kubectl
+elif which kubectl > /dev/null; then
```

Comment:
This is duplicate with lines 32-33 where we use `command -v kubectl`.

Reply:
I don't know why, but github actions doesn't fall into `command -v kubectl` if.
but `which kubectl` does